dev_guide/builds/build_environment.adoc

```diff
@@ -37,14 +43,17 @@ env:
         fieldPath: metadata.name <3>
 ----
 
-<1> the value should be resolved using the following references
-<2> the following `fieldPath` refers to a jsonpath in the `build`
-<3> the jsonpath to the field in the `build`
+<1> `valueFrom` should be resolved using `fieldPath` and `fieldRef`.
+<2> `fieldPath` refers to a `jsonpath` in the `build`.
+<3> `jsonpath` to the field in the `build`.
+
```

Comment:
yeah. i'll let @coreydaley try to fix it, but (2) in particular seems more confusing now since it talks about fieldPath when the field associated w/ the footnote is fieldRef.